Sources/TuistCore/Utils/Functions.swift

```diff
@@ -0,0 +1,36 @@
+/// Produces a getter function for a given key path. Useful for composing property access with functions.
```

Comment:
```swift
import Foundation
```

Reply:
I mentioned this before, these don't actually require `Foundation` as they only the standard lib - do you still want it here even though it's redundant?